src/data/markdown/translated-guides/en/02 Using k6/00 Tutorial/200 Test for performance.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Test for performance
+excerpt: Write thresholds to evaluate performance criteria, then increase load to see how the system performs.
+---
+
+In the previous section, you made a working script to test the endpoint functionality.
+The next step is to test how this system responds under load.
+This requires using the powerful [`options`](/using-k6/options) object, which configures the parts of the test that don't deal with user behavior.
+
+In this tutorial, learn how to:
+- Use [thresholds](/using-k6/thresholds) to assert for performance criteria
+- Configure load increases through [scenarios](/using-k6/scenarios)
+
+These examples build on the script from the previous section.
+
+## Context: meet service-level objectives
+
+To assess the login endpoint's performance, your team may have defined [service level objectives](https://sre.google/sre-book/service-level-objectives/) (SLOs):
+- 99.9% of requests are successful
+- 99% of requests have a latency of 1000ms or less.
+
+The service must meet these SLOs up to peak traffic of 1 request/second.
+
+After you confirm run the peak traffic test, run another test to determine where the system degrades (this is also often done to discover baselines before the organization has an SLOs).
+
+## Assert for performance with thresholds
+
+To codify the SLOs, add [_thresholds_](/using-k6/thresholds) to test that your system performs to its goal criteria.
+Thresholds are exported in options.
```

Comment:
```suggestion
To codify the SLOs, add [_thresholds_](/using-k6/thresholds) to test that your system performs to its goal criteria.
Thresholds are set in options.
```

If we expected zero knowledge for js, using `exported` seems like not a great idea given that users will just not know what that means.

Also we request that they copy code so ... hopefully they will do that.

Reply:
I've tried to make this more explicit and make the example console output cohere in [4375753](https://github.com/grafana/k6-docs/pull/1192/commits/43757539d17e5e3aa74b773122ca692d0effb88a)